src/org/traccar/protocol/WatchProtocolDecoder.java

```diff
@@ -62,15 +67,18 @@ public WatchProtocolDecoder(Protocol protocol) {
             .expression("(.*)")                  // cell and wifi
             .compile();
 
-    private void sendResponse(Channel channel, String id, String index, String content) {
+    private void sendTextResponse(Channel channel, String id, String index, String content) {
```

Comment:
Do we ever send non-text responses? Do we need to rename it?

Reply:
not really, I was also thinking if I should keep it, we can still overload it later if needed. I'll change it back